docs/resources/cfw_eip_protection.md

```diff
@@ -0,0 +1,81 @@
+---
+subcategory: "Cloud Firewall (CFW)"
+---
+
+# huaweicloud_cfw_eip_protection
+
+Manages the protected EIPs under the protect object for CFW service within HuaweiCloud.
+
+~> A protection object (`object_id`) can only create one `huaweicloud_cfw_eip_protection` resource for managing
+protected EIPs.
+
+## Example Usage
+
+```hcl
+variable "object_id" {}
+variable "protected_eips" {
+  type = list(object({
+    id           = string
+    ipv4_address = string
+  }))
+}
+
+resource "huaweicloud_cfw_eip_protection" "test" {
+  object_id = var.object_id
+
+  dynamic "protected_eip" {
+    for_each = var.protected_eips
+
+    content {
+      id          = protected_eip.value["id"]
+      public_ipv4 = protected_eip.value["ipv4_address"]
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `object_id` - (Required, String, ForceNew) The protected object ID.
+  Changing this parameter will create a new resource.
+
+* `protected_eip` - (Required, List) The protected EIP configurations.
```

Comment:
How about rename this field to `protected_eips`

Reply:
`protected_eip` will be better